CLAUDE.md

```diff
@@ -115,6 +140,44 @@ Steps are referenced in `ci-operator/config/` test definitions using multi-stage
 3. Define `pre`, `test`, and `post` phases using existing steps/chains
 4. Run `make validate-step-registry` to check correctness
 
+### Creating Variant Periodic Configurations
+For repositories with many periodic tests, separate them from the main config:
+1. Create `ci-operator/config/<org>/<repo>/<org>-<repo>-<branch>__periodics.yaml`
+2. Include `base_images`, `build_root`, `images`, `promotion`, `releases` sections (copy from main config)
+3. Add only `tests:` entries with `interval:` (periodic scheduling)
+4. Ensure `zz_generated_metadata` has `variant: periodics`
+5. Run `make jobs` to generate Prow jobs in separate `-periodics.yaml` file
+6. This pattern keeps main config focused on PR/postsubmit jobs while isolating scheduled tests
+
+## OpenShift Release Versioning
+
+### Release Branches
+OpenShift uses semantic versioning with minor releases (4.18, 4.19, 4.20, 4.21, 4.22, etc.):
+- **Branch names**: `release-4.21`, `release-4.20`, etc. (or `main`/`master`)
+- **Config files**: One per branch per repository (e.g., `openshift-oauth-server-release-4.21.yaml`)
+- **Release cycle**: New minor versions branch quarterly
```

Comment:
quarterly is not correct, maybe we do not need to make any assumptions here or just very general ones